layer/memory_usage_layer.cc

```diff
@@ -0,0 +1,273 @@
+// Copyright 2020-2021 Google LLC
```

Comment:
AFAIK this is new code, so is 2020 necessary?

Reply:
Parts were copied from the other layers, dated 2020-2021. But there were more copied code in earlier revisions, e.g., the now deleted logic for queue-to-device mapping. So changing to only 2021.